CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+# 107.0.0
+
+In this major release we deleted the deprecated themer classes: `MDCAlertScheme` and `MDCAlertControllerThemer`.
+
+## Breaking changes
+
+Deleted `MDCAlertScheme` and `MDCAlertControllerThemer` class, please use the theming extensions.
```

Comment:
Should we be more specific about what to use instead? How about something like:
"please use `MDCAlertController (MaterialTheming)` extension instead."

Reply:
Done